components/CollectionPictureList.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image'
+import React from 'react'
+import PictureComment from './helpers/PictureComment'
+
+export default function CollectionPictureList({ image }) {
```

Comment:
this is not a list, please rename the component adequately

Reply:
done